core/src/main/resources/hudson/model/Job/configure.properties

```diff
@@ -21,5 +21,5 @@
 # THE SOFTWARE.
 
 name={0} name
-Save=Save All
-Apply=Apply All
\ No newline at end of file
+Save=Save
+Apply=Apply
```

Comment:
Now we don't have tabs anymore, it's all on a single page, so we can revert the labels. *… All* just clarified that you're saving all of the tabs, not just the current one.